nebula3/gclient/net/Session.py

```diff
@@ -18,14 +19,22 @@
 
 
 class Session(object):
-    def __init__(self, connection, auth_result: AuthResult, pool, retry_connect=True):
+    def __init__(
+        self,
+        connection,
+        auth_result: AuthResult,
+        pool,
+        retry_connect=True,
+        retry_times=3,
```

Comment:
we can add another config interval_time to sleep some time between retry executions.

Reply:
Done :)